ReactNativeClient/lib/BaseApplication.js

```diff
@@ -170,6 +170,14 @@ class BaseApplication {
 				continue;
 			}
 
+			// Electron-specific flag used for debugging - ignore it
+			if (arg.includes('--remote-debugging-port')) {
+				// If port number passed also ignore it
+				const argsToIgnore = (nextArg && /^\d+$/.test(nextArg)) ? 2 : 1;
+				argv.splice(0, argsToIgnore);
```

Comment:
Since the port is required in this case, we can shorten it to argv.splice(0, 2)

Reply:
The pattern for the other flags with parameters is '--x y'. This handles the case where the flag has been passed in this form, even tho it is currently unsupported by Electron.